recipes/sctk/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "sctk" %}
+{% set version = "2.4.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/usnistgov/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 43bd2bfe60e0ad04a563ddef025e84fca4c073f8f7c02f056b59fb8e2ec4f022
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
```

Comment:
```suggestion
```
There appears to be no CMakeLists.txt in the upstream project, so CMake is probably not needed.

Reply:
I did this change myself, should've "accepted your suggestion" instead. I would be glad if you could re-check.